sourcebundle/builder.go

```diff
@@ -100,14 +102,15 @@ func NewBuilder(targetDir string, fetcher PackageFetcher, registryClient Registr
 		return nil, fmt.Errorf("invalid target directory: %w", err)
 	}
 	return &Builder{
-		targetDir:               absDir,
-		fetcher:                 fetcher,
-		registryClient:          registryClient,
-		analyzed:                make(map[remoteArtifact]struct{}),
-		remotePackageDirs:       make(map[sourceaddrs.RemotePackage]string),
-		remotePackageMeta:       make(map[sourceaddrs.RemotePackage]*PackageMeta),
-		resolvedRegistry:        make(map[registryPackageVersion]sourceaddrs.RemoteSource),
-		registryPackageVersions: make(map[regaddr.ModulePackage]versions.List),
+		targetDir:                           absDir,
+		fetcher:                             fetcher,
+		registryClient:                      registryClient,
+		analyzed:                            make(map[remoteArtifact]struct{}),
+		remotePackageDirs:                   make(map[sourceaddrs.RemotePackage]string),
+		remotePackageMeta:                   make(map[sourceaddrs.RemotePackage]*PackageMeta),
+		resolvedRegistry:                    make(map[registryPackageVersion]sourceaddrs.RemoteSource),
+		resolvedRegistryVersionDeprecations: make(map[registryPackageVersion]*RegistryVersionDeprecation),
```

Comment:
Slightly verbose-- plus 'resolved' may refer to the RemoteSource values...

```suggestion
		versionDeprecations: make(map[registryPackageVersion]*RegistryVersionDeprecation),
```

Reply:
Could we meet in the middle with `packageVersionDeprecations`? 

Included here: https://github.com/hashicorp/go-slug/pull/59/commits/ea8f696aaf4615b5eed02231bed0038ff283c397